### DISCUSSION TITLE
https://www.transit.land/onestop-id not working

### DISCUSSION THREAD

**Discussion by olivierfischer** [transitland/transitland#393] [Q&A] [answered]
I used all the URLs listed on this page. and get a 404.
Sorry. An error has occured: { "url": "/onestop-id/o-9q9-bart", "statusCode": 404, "statusMessage": "Page not found: /onestop-id/o-9q9-bart", "message": "Page not found: /onestop-id/o-9q9-bart", "stack": "" }

**Comment by drewda** [comment, marked as answer]
Thanks for reporting this issue. It appears that this functionality broke during an update.

We've now fixed it. Try again and let us know if it's working for you, too.
